components/places/android/src/main/java/mozilla/appservices/places/Bookmarks.kt

```diff
@@ -214,6 +214,17 @@ interface ReadableBookmarksConnection : InterruptibleConnection {
      */
     fun getBookmarksWithURL(url: String): List<BookmarkItem>
 
+    /**
+     * Returns the URL for the provided search keyword, if one exists.
+     *
+     * @param The search keyword.
+     * @return The bookmarked URL for the keyword, if set.
+     *
+     * @throws OperationInterrupted if this database implements [InterruptibleConnection] and
+     * has its `interrupt()` method called on another thread.
+     */
+    fun getBookmarkURLForKeyword(keyword: String): String?
```

Comment:
Whoops, one more nit: I think the convention we've been following in Kotlin is `Url`, not `URL`. (Swift is fine, looks like we capitalize it as `URL` there).